Marlin/Marlin_main.cpp

```diff
@@ -5443,7 +5441,7 @@ inline void gcode_M42() {
       if (probe_inverting != deploy_state) SERIAL_PROTOCOLLNPGM("WARNING - INVERTING setting probably backwards");
       refresh_cmd_timeout();
       if (deploy_state != stow_state) {
-        SERIAL_PROTOCOLLNPGM("TLTouch detected");         // BLTouch clone?
+        SERIAL_PROTOCOLLNPGM("TLTouch (NOT BLTouch) detected");         // BLTouch clone?
```

Comment:
Perhaps "BLTouch clone detected" would be better. People are going to start thinking "TLTouch" is another brand they should know about.